src/components/originalSampleProcessing/blockProcessing/BlockProcessingLabwarePlan.tsx

```diff
@@ -105,6 +94,7 @@ const BlockProcessingLabwarePlan = React.forwardRef<HTMLDivElement, BlockProcess
     const { requestError, layoutPlan } = current.context;
     const { layoutMachine } = current.children;
     const { setFieldValue, values } = useFormikContext<BlockFormData>();
+    const [disableRepNumber, setDisableRepNumber] = useState<boolean>(false);
```

Comment:
We can use values.plans[rowIndex].replicateNumber instead of this which avoids managing another react state.